test/controllers/categories_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+
+class CategoriesControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @category = categories(:one)
```

Comment:
dont forget also change test when you make changes with multiple categories

Reply:
done